src/server/components/lists/helpers.ts

```diff
@@ -160,6 +161,20 @@ export const getCountryLawyerRedirectLink = (() => {
   };
 })();
 
+export const getCountryFuneralDirectorsRedirectLink = (() => {
+  const pagesByCountry = mapKeys(fcdoFuneralDirectorsByCountry, (_, key) =>
+    lowerCase(key)
+  );
+
+  return (countryName: CountryName): string => {
+    return get(
+      pagesByCountry,
+      lowerCase(countryName),
+      "https://www.gov.uk/government/collections/funeral-directors-worldwide-list"
+    );
+  };
+})();
```

Comment:
oh.. I didn't realise this is how we were getting the lawyers link too. imo this is more complex than what it's worth. not that there's anything wrong with currying/FP.. but the returned function can't be tested independently of it's wrapping function ¯\_(ツ)_/¯. lodash.get is pretty useless in this case as well. 


```suggestion
const funeralDirectorPagesByCountry = mapKeys(fcdoFuneralDirectorsByCountry, (_, key) =>
     lowerCase(key)
   );
function getFuneralDirectorRedirectLink(country: CountryName) {
  const DEFAULT_LINK = "https://www.gov.uk/government/collections/funeral-directors-worldwide-list"
  return funeralDirectorPagesByCountry[country?.toLowerCase()] ?? DEFAULT_LINK;
}
```